mobile/src/store/course-modules/common.ts

```diff
@@ -1,5 +1,6 @@
 enum ActionType {
   GET_COURSE_MODULES = 'course-modules/get-course-modules',
+  GET_MODULE_BY_ID = 'course-module/get-module-by-id',
```

Comment:
I think better reduce it to a general: _course-modules_ or _course-module_
```suggestion
  GET_MODULE_BY_ID = 'course-modules/get-module-by-id',
```

Reply:
yes, I agree. 
It was late, didn't pay attention